tests/test_termcolor.py

```diff
@@ -25,6 +25,8 @@ def setup_module() -> None:
 def test_basic(monkeypatch: pytest.MonkeyPatch) -> None:
     # Arrange
     monkeypatch.setattr("sys.stdout.isatty", lambda: True)
+    monkeypatch.setattr("sys.stdout.fileno", lambda: 1)
+    monkeypatch.setattr(os, "isatty", lambda fd: True)
```

Comment:
Nitpick: please could you put these three statements in alphabetical order? And the same below.

Reply:
Sure!

Reply:
Fixed!